packages/ExternalLink/stories/examples/Showcase.js

```diff
@@ -9,7 +9,7 @@ const externalLinkProps = () => ({
   ariaText: text("ariaText", ""),
   hasNoUnderline: boolean("hasNoUnderline", false),
   href: text("href", "http://www.wegalvanize.com"),
-  width: number("Inline width px value", "120"),
+  maxWidth: number("Inline maxWidth", "120"),
```

Comment:
`maxWidth` makes a better example than `width`, otherwise you can end up with a bit empty gap between the (short) hyperlink label and the external link icon.